Server.MirForms/Database/RecipeInfoForm.cs

```diff
@@ -748,5 +771,39 @@ private void ClearForm()
             IngredientDura6TextBox.Text = string.Empty;
         }
         #endregion
+
+        #region Search Box
+        private void RecipeSearchBox_KeyDown(object sender, KeyEventArgs e)
+        {
+            // Trigger search only on Enter key press
+            if (e.KeyCode == Keys.Enter)
+            {
```

Comment:
Doesn't look to have been corrected yet. As per my previous comments..

Doesn't really make sense that this is the only search box you need to press enter on.

Remove all this load logic in to its own method, store the recipeFiles in a public list. then filter on this public list the same way it works in the other forms you've updated.